kubectl-rook-ceph.sh

```diff
@@ -226,14 +227,176 @@ function path_cm_rook_ceph_operator_config() {
 }
 
 ####################################################################################################
-# 'kubectl rook-ceph mon-endpoints' commands
+# 'kubectl rook-ceph mons' commands
 ####################################################################################################
 
+function run_mons_command () {
+  if [ "$#" -ge 1 ] && [ "$1" = "reset-quorum" ]; then
+    shift # remove the subcommand from the front of the arg list
+    run_reset_quorum "$@"
+  else
+    fetch_mon_endpoints "$@"
+  fi
+}
+
 function fetch_mon_endpoints() {
   end_of_command_parsing "$@" # end of command tree
   KUBECTL_NS_CLUSTER get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data' | tr -d '"' | tr -d '=' | sed 's/[A-Za-z]*//g'
 }
 
+
+wait_for_pod_of_deployment_to_be_running() {
+  echo -e "\nTODO: waiting for the pod from deployment \"rook-ceph-mon-$good_mon-debug\" to be running"
+  sleep 5
+}
```

Comment:
```suggestion
wait_for_pod_of_deployment_to_be_running() {
  echo -e "\nTODO: waiting for the pod from deployment \"rook-ceph-mon-$good_mon-debug\" to be running"
  sleep 5
}
```

instead, can we do something like 
```
kubectl wait deployment -n <ns> <deployment-name> --for condition=Available=True --timeout=90s
```

Reply:
Yeah, the sleep was just a hack :)